GOVERNANCE.md

```diff
@@ -0,0 +1,110 @@
+StackStorm has joined LinuxFoundation in 2019.
+With the new Neutral Home we invite everyone: organizations, teams and individuals relying on StackStorm Automation in their operations to support the project by giving back and participating in development as well as influence on its future under the Open Governance.
+
+# StackStorm Governance
+This document defines governance policies for the StackStorm project.
+It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2019/10/StackStorm-Technical-Charter-20191004.pdf) for StackStorm within LinuxFoundation.
+
+## Technical Steering Committee (TSC)
+The Technical Steering Committee is a group of Maintainers (Committers) who have earned the ability to modify (“commit”) sourcecode, documentation or other artifacts.
+This group is responsible for all technical oversight of StackStorm as Open Source Project.
+
+### Maintainer Roles
+The current list of maintainers is published and updated in [OWNERS.md](OWNERS.md).
+StackStorm uses a three-tiered system of Maintainer roles:
+* [Leaders](OWNERS.md#leaders-)
+  * Head of Technical Steering Committee (TSC).
+  * Responsible for Project Strategy, External Relations, Organizational aspects, coordinating Events, Partnerships.
+  * Receive **three votes** in the [conflict resolution and voting process](#conflict-resolution-and-voting) described below.
+* [Senior Maintainers](OWNERS.md#senior-maintainers-)
+  * Have the most in-depth experience with the StackStorm project and are expected to have the knowledge and insight to lead the project's future, growth, standards and improvement.
+  * Oversee the process for adding new maintainers and provide guidance, help and sharing their experience with the standard maintainers.
+  * Have full owner access to all the resources and platforms that sustain StackStorm project.
+  * Receive **two votes** in the voting process.
+* [Maintainers](OWNERS.md#maintainers-)
+  * Have good experience with the StackStorm codebase, expected to provide significant value to the project, helping it grow, improve and succeed.
+  * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
```

Comment:
Should maintainers also have access to CI so they can force rebuild if necessary?

Reply:
:+1: Added CI/CD to be explicit.